tests/test_dialects.py

```diff
@@ -833,19 +1038,22 @@ def test_hive(self):
         )
         self.validate(
             "DATE_SUB('2020-01-01', 1)",
-            "DATE_FORMAT(DATE_ADD('DAY', 1 * -1, DATE_PARSE(SUBSTR('2020-01-01', 1, 10), '%Y-%m-%d')), '%Y-%m-%d')",
+            "DATE_ADD('DAY', 1 * -1, '2020-01-01')",
             read="hive",
             write="presto",
         )
         self.validate(
             "DATE_ADD('2020-01-01', 1)",
-            "DATE_FORMAT(DATE_ADD('DAY', 1, DATE_PARSE(SUBSTR('2020-01-01', 1, 10), '%Y-%m-%d')), '%Y-%m-%d')",
+            "DATE_ADD('DAY', 1, '2020-01-01')",
```

Comment:
is this test correct? have you tested this works? in presto don't you need an actual date type not a varchar

Reply:
You are right this doesn't work. I will review this. I originally ran all the conversions manually but that was before the mixed type refactor so I need to do it again.